ts/torch_handler/distributed/base_pippy_handler.py

```diff
@@ -0,0 +1,28 @@
+"""
+Base default handler to load large models using PyTorch Native PiPPy.
+"""
+import os
+from abc import ABC
+
+import torch
+
+from ts.handler_utils.distributed.pt_pippy import initialize_rpc_workers
+from ts.torch_handler.base_handler import BaseHandler
+
+
+class BasePippyHandler(BaseHandler, ABC):
+    """
+    Base default handler to set up rpc workers for PiPPy large model inference
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.initialized = False
+
+    def initialize(self, ctx):
+        super().__init__()
+        self.local_rank = int(os.environ["LOCAL_RANK"])
+        self.world_size = int(os.environ["WORLD_SIZE"])
+        n_devs = torch.cuda.device_count()
+        self.device = self.local_rank % n_devs
+        initialize_rpc_workers(self.local_rank, self.world_size, ctx)
```

Comment:
you can add:
self.model = get_pipline_driver(self.model, self.world_size, ctx)

Reply:
I am wondering if leaving it in the custom handler may provide more clarity to user what is happening?